experimental/demoprobe/generate.go

```diff
@@ -123,3 +121,5 @@ func DemoReport(nodeCount int) report.Report {
 
 	return r
 }
+
+func newu64(value uint64) *uint64 { return &value }
```

Comment:
This is unfortunately a function of necessity: there's no way to take the address of a primitive type.